lib/components/app/app-menu.tsx

```diff
@@ -180,11 +180,13 @@ class AppMenu extends Component<
     return (
       <>
         <div
+          aria-expanded={isPaneOpen}
           aria-label={
             isPaneOpen
               ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
               : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
           }
+          aria-roledescription="button"
           className="app-menu-icon"
           onClick={this._togglePane}
           onKeyDown={this._togglePane}
```

Comment:
Won't hold this against this PR, but this traps the tab focus to the component (pressing tab or any key expands the side pane). Noting this as a new issue.